Traffic Policer/Main.cs

```diff
@@ -63,12 +63,12 @@ public override void Initialize()
 
         }
         //Dependencies
-        internal static Version Albo1125CommonVer = new Version("6.6.2.0");
+        internal static Version Albo1125CommonVer = new Version("6.6.3.0");
         internal static Version MadeForGTAVersion = new Version("1.0.1493.1");
         internal static float MinimumRPHVersion = 0.51f;
-        internal static string[] AudioFilesToCheckFor = new string[] { "LSPDFR/Police Scanner/Traffic Policer Audio/OTHER_UNIT_TAKING_CALL/OTHER_UNIT_TAKING_CALL_01.wav", "LSPDFR/Police Scanner/Traffic Policer Audio/Crimes/CRIME_DUI_01.wav" };
+        internal static string[] AudioFilesToCheckFor = new string[] { "lspdfr/audio/scanner/Traffic Policer Audio/OTHER_UNIT_TAKING_CALL/OTHER_UNIT_TAKING_CALL_01.wav", "lspdfr/audio/scanners/Traffic Policer Audio/Crimes/CRIME_DUI_01.wav" };
```

Comment:
`audio/scanners/` is incorrect, change to `audio/scanner` as like the first path

Reply:
> `audio/scanners/` is incorrect, change to `audio/scanner` as like the first path

Whoops! Just a little typo on his part lol.